core/src/main/java/tc/oc/pgm/filters/query/GoalQuery.java

```diff
@@ -7,8 +7,14 @@
  * have filters that respond specifically to goal queries, but currently we do not.
  */
 public class GoalQuery extends MatchQuery {
```

Comment:
Now that this is used we should probably include a matching interface in `tc.oc.pgm.api.filter.query` as every(?) other query seems to have their own counterpart.

Reply:
I agree, but it is out of scope for this pr